WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -26,21 +26,23 @@ final class StoreOnboardingViewHostingController: UIHostingController<StoreOnboa
         super.viewDidLoad()
 
         configureNavigationBarAppearance()
-        reloadTasks()
+        Task {
+            await reloadTasks()
+        }
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        reloadTasks()
-    }
-
-    private func reloadTasks() {
         Task {
-            await viewModel.reloadTasks()
+            await reloadTasks()
         }
     }
 
+   func reloadTasks() async {
```

Comment:
nit: do we want to mark this as `@MainActor` to make sure it's run on the main thread?

Reply:
Done in b73b2c22809b5ab0ccbe1ee093f65bfb61ebbfb7